javaparser-core-testing/src/test/java/com/github/javaparser/printer/DefaultPrettyPrinterTest.java

```diff
@@ -666,4 +666,36 @@ void testPrinterWithEclipseImportOrdering() {
 
         assertEqualsStringIgnoringEol(expectedCode, actualCode);
     }
+
+    @Test
+    void testArrayOfAnnotationsFormat() {
```

Comment:
Can you add a test case that does not activate the printing of annotations on several lines?

Reply:
Done